src/tests/ftest/control/ms_resilience.py

```diff
@@ -183,6 +183,10 @@ def launch_servers(self, resilience_num):
                 },
         }
         self.start_servers(server_groups)
+
+        # Give SWIM some time to stabilize before we start killing stuff (DAOS-9116)
```

Comment:
this looks like a workaround for a legit problem, of something wrong happening if servers are killed too quickly. is it the right thing to hide such issue?

Also 5 seconds is not sufficient in some cases for swim to stabilize

Reply:
@dmiter is 5 seconds going to be enough time to get swim to ping nodes in this case, or should the wait time be bumped higher?

Reply:
SWIM pings once per second a random rank. Therefore it will need at least <count of ranks> * 1 sec to ping all of them.